test/index.js

```diff
@@ -1,9 +1,9 @@
 jest.mock('../plugin/src/helpers/utils', () => {
   return {
-    ...(jest.requireActual('../plugin/src/helpers/utils')),
-    isNextAuthInstalled: jest.fn()
+    ...jest.requireActual('../plugin/src/helpers/utils'),
```

Comment:
Extra noise in this file due to my autosave lint plugin. Good news is this file now follows our eslint and prettier rules :sweat_smile: